packages/client/src/Components/JsonCreator.jsx

```diff
@@ -18,7 +18,15 @@ class JsonCreator extends Component {
       selectedSchema: ''
     };
   }
-
+  
+  /*
+  ** avoid loading of flatpickr load in Message template screen for admin
```

Comment:
@sanjayradadiya - could you expand this comment a little please?

(Or reply to this comment with more explanation if that's easier :-) )

Reply:
flatpickr calendar uses the "window" object during initialization in the library itself which is generated issues for admin message template preview. avoid this issue window.flatpickr object set to undefined in JsonCreator component.

Reply:
Aah, I see, thanks.